Policies/Kubernetes/allowed-proc-mount-types/rego/src.rego

```diff
@@ -0,0 +1,21 @@
+package k8sazureprocmount
+
+violation[{"msg": msg, "details": {}}] {
+    c := input_containers[_]
+    not input_proc_mount_type_allowed(c)
+    msg := sprintf("ProcMount type is not allowed, container: %v. Allowed procMount types: %v", [c.name, input.parameters.procMountType])
+}
+
+input_proc_mount_type_allowed(c) {
+    input.parameters.procMountType == c.securityContext.procMount
```

Comment:
Can you check if Azure Policy parameter for procMountType allows values in case-insensitive way? i.e., are both 'default' and 'Default' allowed? If yes, can you check if REGO equals also honors the case-insensitive comparison?

Reply:
The Azure Policy parameter only allows 'Default' the lowercase example is not allowed. I've update the rego to honor the case-insensitive option to be more robust however, in case this changes in the future.